dkarlovi/symfony-flex-docker/0.1/bin/dphp

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Because this is not part of the project, you use a [preconfigured utility Docker container](https://github.com/dkarlovi/php-toolbox) which carries all the QA and such tools so your `vendor/` folder is not bloated (with this approach, I've reduced my `vendor/` folder size by 50%), also removing some nasty conflicts in my Composer dependencies.

This way, you build your app without development dependencies (again, there is no "environment-specific environment", you build once and run everywhere) and ship your development tools in a separate container, managed elsewhere.

Also, the toolbox container invocation script is setup so that you can use it like a custom PHP interpreter in PhpStorm, ie. you point PhpStorm directly to this file as a PHP interpterer and it automatically can run your tests and tools INSIDE your local container cluster.